README.md

```diff
@@ -11,13 +11,15 @@ This is the repository for the Open Source Vulnerability schema, which is curren
 - [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
 - [AlmaLinux](https://github.com/AlmaLinux/osv-database)
 - [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+- [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
 - [OSV.dev maintained converters](https://github.com/google/osv.dev#current-data-sources)
```

Comment:
Since this is not alphabetically ordered, let's put it at the end
```suggestion
- [OSV.dev maintained converters](https://github.com/google/osv.dev#current-data-sources)
- [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
```

Reply:
[Previous PR](https://github.com/ossf/osv-schema/pull/157/files) didn't add the Haskell reference as the latest element on the list. @oliverchang, would you mind clarifying this point? Thanks!

Reply:
The OSV.dev ones are a weird one that we typically leave at the end. This order is fine, thanks!